src/utils/getVisibleTodos.ts

```diff
@@ -0,0 +1,13 @@
+import { Todo } from '../types/Todo';
+import { FilterBy } from '../types/FilterBy';
+
+export const getVisibletodos = (todos: Todo[], filterBy: FilterBy): Todo[] => {
+  switch (filterBy) {
+    case FilterBy.ACTIVE:
+      return todos.filter(todo => !todo.completed);
+    case FilterBy.COMPLETED:
+      return todos.filter(todo => todo.completed);
+    default:
+      return todos;
```

Comment:
Maybe you can use switch-case inside filter()

Reply:
Good idea, but in this case, I should save a boolean in filterType. I need strings because I use its when creating the filter